src/term.c

```diff
@@ -4511,6 +4555,15 @@ check_termcode(
 # endif
 			}
 		    }
+		    /* Third row is xterm compatibility test */
+		    else if (row_char == '3') {
+			if (col != 1) {
```

Comment:
Could you move the last `{` to the next line? (also for the above line.)

Reply:
updated. thanks!